src/main/java/com/dreamhorizon/enchantments/utils/NumberUtil.java

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Rename package to "util" so it's like core.

Reply:
Renamed